search/apps.py

```diff
@@ -0,0 +1,16 @@
+from django.apps import AppConfig
+from django.db.models.signals import post_migrate
+
+from .utils import create_index_table
+
+
+def create_index(sender, **kwargs):
+    create_index_table()
```

Comment:
Humm, you mean why not registering directly create_index_table` to the signal instead? If yes, the answer is that the function is also used in the management command, so it needs a neutral signature.